### ISSUE TITLE
Confirm v1.3 visual UI glitches as bugs?

### ISSUE THREAD

**Issue by mackonsti** [jief666/BootloaderChooser#9]
Hello @jief666 thanks again for your fixes in release v1.3

N.B. Please kindly modify the release title as you have added in both places the same comment (instead of the release version) see in https://github.com/jief666/BootloaderChooser/releases

I have tested the release and with a non-existent `BLC.plist` file (renamed my old one) to "start clean" and I have the following to comment:

**1. There is no volume name appearing in the square brackets, anymore?**

I remember that you used to show the volume name of the EFI partition inside the square [ brackets ] is it not the case, anymore? I am using only Clover on the target machine, not OpenCore. See photo:

![IMG_9950](https://github.com/jief666/BootloaderChooser/assets/1313613/504ee1a7-99f9-43d0-b91f-bb2fb6491a78)

Here is my disk structure, EFI partition is on disk1 for some reason (renamed to "NUC" long time ago):

```
% diskutil list

/dev/disk0 (internal, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:     FDisk_partition_scheme                        *500.1 GB   disk0
   1:                  Apple_HFS Samsung EVO             500.1 GB   disk0s1

/dev/disk1 (internal, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:      GUID_partition_scheme                        *120.0 GB   disk1
   1:                        EFI NUC                     209.7 MB   disk1s1
   2:                 Apple_APFS Container disk2         119.8 GB   disk1s2

/dev/disk2 (synthesized):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:      APFS Container Scheme -                      +119.8 GB   disk2
                                 Physical Store disk1s2
   1:                APFS Volume Samsung NUC - Data      35.7 GB    disk2s1
   2:                APFS Volume Preboot                 82.6 MB    disk2s2
   3:                APFS Volume Recovery                529.9 MB   disk2s3
   4:                APFS Volume VM                      1.1 MB     disk2s4
   5:                APFS Volume Samsung NUC             11.3 GB    disk2s5
```

**2. Rebooting PC from Options, works as expected.**

FYI

**3. When entering Options, setting a Timeout value in seconds works as before.**

But the input box is kind of misleading; there is no cursor flashing, is this expected? I think Clover does that i.e. press ENTER once to access the value to modify, then modify value (with backspace or delete, too) then press ENTER once more to exit edit mode (and be able to navigate with UP/DOWN arrows etc.)

In this process, there is no cursor appearing or blinking -- please confirm if this expected.

**4. There is a UI glitch when showing the path of the default volume.**

![IMG_9952](https://github.com/jief666/BootloaderChooser/assets/1313613/1feddfa8-d173-4ad7-80dd-c6528f45c02e)

![IMG_9951](https://github.com/jief666/BootloaderChooser/assets/1313613/ff0ef68b-7251-4337-957d-b0e6ba376ca9)

This is the case if this path is over 1 line in length... see how the display of `Timeout: 3` is overlaid on the path.

As a quick fix, I strongly suggest you move that as a last item as there won't be any options to highlight after that. I mean, reverse the menu listed, start with `Return`, then `Save debug log`, then `Timeout` and finally `Default volume`. That way, the volume path can take 2 lines without affecting other Options entries...

**5. The debug log option (true/false) writes a key value in PLIST as "debug"**

All PLIST keys have a capitalised name, only this one is all-lowercase. The fix would require to make it conforming to the rest of it and save and read as `Debug` keyname. This is what's recorded in PLIST:

```
<key>debug</key>
<string>false</string>
```

Thanks for confirming 🙏

**Comment by jief666** [comment]
3. Yes, no cursor. I don't know why. The text version of menus imported from Clover is probably outdated as Clover user don't use text since a while, I guess.
Same a line too long. It's not handled by the text menu module.
I don't I'll fix that. Too much time for little gain...

All the rest is fixed.

**Comment by mackonsti** [comment]
Merci @jief666 this is great, thanks for taking the time to fix them!

**Comment by mackonsti** [comment]
@jief666 please have a look at your releases page https://github.com/jief666/BootloaderChooser/releases and when you have time, perhaps fix the release text of 1.3 😄
